lib/shutil_custom/__init__.py

```diff
@@ -6,35 +6,31 @@
 except ImportError:
     from shutil import Error as SameFileError
 
-try:
-    O_BINARY = os.O_BINARY  # Windows
-except AttributeError:
-    O_BINARY = 0
-
 
-BUFFER_SIZE = 128*1024
+BUFFER_SIZE = 1024 * 1024 if os.name == 'nt' else 16 * 1024
 
 
 def copyfile_custom(src, dst):
     """Copy data from src to dst."""
     def special_file(fn):
         try:
             st = os.stat(fn)
-        except OSError:  # File most likely does not exist
+        except OSError:
+            # File most likely does not exist
             pass
         else:
             if stat.S_ISFIFO(st.st_mode):
-                raise SpecialFileError("`%s` is a named pipe" % fn)
+                raise SpecialFileError("{!r} is a named pipe".format(fn))
 
-    with open(src, 'rb') as fin, open(dst, 'wb') as fout:
+    with open(src, 'rb') as fsrc, open(dst, 'wb') as fdst:
         if _samefile(src, dst):
-            raise SameFileError("`%s` and `%s` are the same file" % (src, dst))
+            raise SameFileError("{!r} and {!r} are the same file".format(src, dst))
 
         special_file(src)
         special_file(dst)
 
-        while True:
-            data = fin.read(BUFFER_SIZE)
-            if data == '':
+        while 1:
```

Comment:
Style preference only, but I think `while True:` is clearer than `while 1:`

Reply:
Changed 👌